website/docs/r/policy_lb_https_monitor_profile.html.markdown

```diff
@@ -0,0 +1,84 @@
+---
+subcategory: "Load Balancer"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_lb_https_monitor_profile"
+description: A resource to configure a LBHttpsMonitorProfile.
+---
+
+# nsxt_policy_lb_https_monitor_profile
+
+This resource provides a method for the management of a LBHttpsMonitorProfile.
+
+This resource is applicable to NSX Global Manager, NSX Policy Manager and VMC.
+
+## Example Usage
+
+```hcl
+resource "nsxt_policy_lb_https_monitor_profile" "test" {
+  display_name          = "test"
+  description           = "Terraform provisioned LBHttpsMonitorProfile"
+  request_body          = "test"
+  request_method        = "POST"
+  request_url           = "test"
+  request_version       = "HTTP_VERSION_1_1"
+  response_body         = "test"
+  response_status_codes = 2
```

Comment:
I believe this should be a list, and it would be helpful to show values that make sense - 200 here, and a reasonable port value below